README.md

```diff
@@ -33,14 +33,15 @@ Prerequisites
     $ cd libtock-c
     ```
 
-1. The main requirement to build the C applications in this repository
-   is a cross compiler for embedded targets. Currently, the only
-   supportetd target is ARM Cortex M, so you will need an
-   `arm-none-eabi` toolchain.
+1. The main requirement to build the C applications in this repository is having
+   cross compilers for embedded targets. You will need an `arm-none-eabi`
+   toolchain for Cortex-M targets, and `riscv64-unknown-elf` toolchain for
+   RISC-V targets.
 
    MacOS:
    ```
    $ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+   $ brew install riscv-gnu-toolchain --with-multilib
    ```
 
    Ubuntu (18.04LTS or later):
```

Comment:
I can't build any apps on this branch because there aren't instructions for installing the riscv toolchain for Linux. 

I tried `sudo apt install gcc-riscv64-unknown-elf`, which got me past my initial error about the command not being found, but led to:

```
(.venv) hudson: ~/libtock-c/examples/blink (riscv-for-all) $ make
  CC        ../../libtock/internal/alarm_internal.c
In file included from <command-line>:31:
./../../support/warning_header.h:6:10: fatal error: stdio.h: No such file or directory
    6 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [../../TockLibrary.mk:149: ../../libtock/build/rv32imac/alarm_internal.o] Error 1
```

Do we have to use the siFive toolchain?

Reply:
I'm not sure, I need help here.